File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -7,7 +7,7 @@ documentation: UG
 ---
 # Working with Table Of Contents
 
-[Table of contents](https://support.microsoft.com/en-gb/office/insert-a-table-of-contents-882e8564-0edb-435e-84b5-1d8552ccf0c0?redirectSourcePath=%252fen-us%252farticle%252fCreate-a-table-of-contents-or-update-a-table-of-contents-eb275189-b93e-4559-8dd9-c279457bfd72#__create_a_table ) (TOC) is used to provide an outline of the Word document. By default table of contents will be created automatically from heading styles. 
+[Table of contents](https://support.microsoft.com/en-gb/office/insert-a-table-of-contents-882e8564-0edb-435e-84b5-1d8552ccf0c0?redirectSourcePath=%252fen-us%252farticle%252fCreate-a-table-of-contents-or-update-a-table-of-contents-eb275189-b93e-4559-8dd9-c279457bfd72#__create_a_table) (TOC) is used to provide an outline of the Word document. By default table of contents will be created automatically from heading styles. 
```

Comment:
Is this correct?

Reply:
Removed extra space between the link and the bracket.